github-git/2-identify-issue.md

```diff
@@ -0,0 +1,121 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+(identify-issue)=
+# Identify an issue to work on 
+
+Open issues are where developers track bugs, new features, or other improvements needed in the repo. Whether you have a specific fix in mind or are looking for ideas, you should start by exploring existing open issues.
+
+In this lesson you will create or comment on an issue that you'd like to fix in GitHub repository that you don't own. 
+
+:::{admonition} Activity: Create (or find) an issue for a bug that you'd like to fix 
```

Comment:
Along the same lines as @jsdodge 's comment on this line, this comes a little bit out of order for me. So in the flow of the lesson we have just finished getting to know the repository. Presumably in reality we are starting from the place of already using the package and noticing something wrong as step -1, and while I understand the idea behind the dummy repository, it seems odd to start this section with "go look for a bug." maybe this should come at the end of this document?